db/migrate/20250117112840_create_professional_body_membership.rb

```diff
@@ -0,0 +1,15 @@
+class CreateProfessionalBodyMembership < ActiveRecord::Migration[7.2]
+  # rubocop:disable Rails/ThreeStateBooleanColumn
```

Comment:
I think in this case the 3 state boolean is wrong - exam_taken is a true/false radio, so it is always known and cannot be null